index.textile

```diff
@@ -1,43 +1,27 @@
-h2. BlocklyProp help overview
+h3. Let's get started!
 
 p(short). BlocklyProp help overview
 
- <div class="col-sm-4 col-xs-12">
+<div class="col-sm-4 col-xs-12">
```

Comment:
I've spent enough hours writing textile pages already. The Java library we use has some weird variations on the standard, so I had to look in the code for some of them.